.flake8

```diff
@@ -5,8 +5,12 @@
 #   https://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes
 
 ignore =
+    # E203: whitespace before ':'
+    E203,
     # E402: module level imports at top of file
     E402,
```

Comment:
Iris also has `    # E231: missing whitespace after ',', ';', or ':'` here, should this also be included ore is there a reason for the difference?

Reply:
I checked and `E203` is recommended by Black but `E231` is not. I'd be happy to add more ignores if you think consistency with Iris is important?

Reply:
Looks like this issue has been addressed here: https://github.com/psf/black/issues/1289, I'd say we're probably good as is.